upgrade/upgrade-1.4.0.php

```diff
@@ -143,6 +143,8 @@ function upgrade_module_1_4_0($module)
 
     return $result
         && $module->registerHook('displayAdminOrderLeft')
+        && $module->registerHook('displayAdminOrderMainBottom')
```

Comment:
This hook is in your list for PS 1.7, don't you want to check the current version?

Reply:
Not needed, it will be added be INSERT IGNORE on PrestaShop upgrade so no conflict.
If a merchant upgrade from an older version, no need to register hook manualy

Reply:
That's a fair point, but in that case why don't we register it for new merchants too? You should have the same behavior for everyone or you will encounter issues depending on the life of the module on the shop.

Reply:
Need to check all hooks in upgrade script of PrestaShop because some are added without INSERT IGNORE and cause crash of PrestaShop upgrade (mainly fist versions of 1.7)